src/dp_abstraction/aca_dataplane_ovs.cpp

```diff
@@ -226,6 +273,13 @@ int ACA_Dataplane_OVS::update_port_state_workitem(const PortState current_PortSt
               current_PortConfiguration.vpc_id(), generated_port_name,
               found_tunnel_id, culminative_dataplane_programming_time);
 
+      if (found_auxgateway.aux_gateway_type() == ZETA) {
+        ACA_LOG_INFO("%s", "AuxGateway_type is zeta!\n");
+        // Delete the zeta settings of vpc
+        overall_rc = ACA_Zeta_Programming::get_instance().delete_zeta_config(
```

Comment:
what happen when there is multiple ports (from the same VPC) sharing this zeta configuration? when one of the port is deleted, we cannot simply delete the configuration because it will affect the other port which is not deleted.